plugins/sinks/console/sink.go

```diff
@@ -25,13 +25,12 @@ func (s *Sink) Sink(ctx context.Context, config map[string]interface{}, out <-ch
 }
 
 func (s *Sink) Process(value interface{}) error {
-	data, err := json.Marshal(value)
+	jsonBytes, err := json.Marshal(value)
 	if err != nil {
 		fmt.Println(fmt.Sprint(value))
```

Comment:
Btw why do we contain the error here? 🤔

Reply:
Haha this is not my change although I should have cleaned it up, I only renamed the variable while resolving merge conflicts. Let me fix it.